executor/src/test/java/org/k8loud/executor/kubernetes/BaseTest.java

```diff
@@ -6,9 +6,13 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mock;
-import org.mockito.Mockito;
 import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
 
+import static org.mockito.Mockito.when;
+
+@MockitoSettings(strictness = Strictness.LENIENT)
```

Comment:
Dont use it. Stay with the previous way when you use lenient for one operation

This annotation leads to using `lenient()` in all mockito `when()` calls which is not recommended - it makes tests less understandable.

We are not using lenient to avoid creating unnecessary mockito `when` calls.